### ISSUE TITLE
Run pylint on the event manager

### ISSUE THREAD

**Issue by mathstuf** [uzbl/uzbl#265]
Lots of things to clean up in there. I'd recommend tackling each error class as a separate branch.

**Comment by keis** [comment]
Agreed. Following the community conventions makes it much easier to jump right in and change some code. However when using pylint we must be aware it sometimes gives false positives and in general is just very picky.

e.g
defining functions from a high-order return trips the invalid constant name check.
shadowing builtins and redefining local variables is fine in moderation.
requiring docstrings on every single one liner function is a bit excessive, however module and class level docstrings really should be included.
